delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/config/CompositeViewsConfig.scala

```diff
@@ -40,6 +45,7 @@ import scala.concurrent.duration.FiniteDuration
   */
 final case class CompositeViewsConfig(
     sources: SourcesConfig,
+    blazegraph: BlazegraphInstanceConfig,
```

Comment:
`access: BlazegraphAccess` ?
We do have `DatabaseAccess` already

Reply:
I used `blazegraphAccess` for the field name since `access` on it's own kind of begs the question "access to what?". Works like this?